web/app.js

```diff
@@ -2759,6 +2780,18 @@ function onKeyUp(evt) {
 function onKeyDown(evt) {
   this._isCtrlKeyDown = evt.key === "Control";
 
+  if (
+    this.editorUndoBar?.isOpen &&
+    evt.keyCode !== 9 &&
+    evt.keyCode !== 16 &&
+    (getActiveOrFocusedElement() !== this.editorUndoBar.undoButton ||
+      !(evt.keyCode === 13 || evt.keyCode === 32))
+  ) {
+    // Hide undo bar on keypress except for Shift, Tab , Shift+Tab
+    // Also avoid hiding if the undo button is triggered
```

Comment:
To aid readability, please remember to end sentences in comments with a period.
```suggestion
    // Hide undo bar on keypress except for Shift, Tab, Shift+Tab.
    // Also avoid hiding if the undo button is triggered.
```